admin/operational.js

```diff
@@ -71,6 +72,32 @@ function testConnection () {
     });
 }
 
+function setActualTime () {
+    utils.getDeviceId(function (id) {
+        var cloud = Cloud.init(logger, id);
+        cloud.getActualTime(function (time) {
+            var r = 0;
```

Comment:
Ok. So leave 2 instances of process.exit, but introduce constants for special values: 0 and 1.